costmap_2d/include/costmap_2d/observation_buffer.h

```diff
@@ -159,6 +160,7 @@ class ObservationBuffer
   boost::recursive_mutex lock_;  ///< @brief A lock for accessing data in callbacks safely
   double obstacle_range_, raytrace_range_;
   double tf_tolerance_;
+  double start_clearing_range_;
```

Comment:
reaname to min_raytrace_range_, update raytrace_range to be max_raytrace_range_, and let's declare all of them on the same line above: "double obstacle_range_, min_raytrace_range_, max_raytrace_range_".